packages/discovery-react-components/package.json

```diff
@@ -58,6 +58,9 @@
     "react-virtualized": "^9.21.1"
   },
   "jest": {
+    "transformIgnorePatterns": [
+      "/!node_modules\\/react-virtualized/"
```

Comment:
do we want to include all node modules? or just this library?

Reply:
This tells jest to not try to do a transform on this library, since it is already being included as an ES module. If it were commonjs, then it _would_ need to be transformed.